integration/check_test.go

```diff
@@ -84,7 +84,7 @@ func (s *SkopeoSuite) TestVersion(c *check.C) {
 func (s *SkopeoSuite) TestCanAuthToPrivateRegistryV2WithoutDockerCfg(c *check.C) {
 	out, err := exec.Command(skopeoBinary, "--docker-cfg=''", "--username="+s.regV2WithAuth.username, "--password="+s.regV2WithAuth.password, "inspect", fmt.Sprintf("docker://%s/busybox:latest", s.regV2WithAuth.url)).CombinedOutput()
 	c.Assert(err, check.NotNil, check.Commentf(string(out)))
-	wanted := "Error: image busybox not found"
+	wanted := "Invalid status code returned when fetching manifest 401"
 	if !strings.Contains(string(out), wanted) {
```

Comment:
not sure about this, those error strings aren't part of the API and are just returned in one place. I don't want to use typed errors in tests cause otherwise if I have a bug in the error type I won't catch it if using the error type itself